services/system_undelete.proto

```diff
@@ -29,18 +46,43 @@ option java_multiple_files = true;
 import "basic_types.proto";
 
 /**
- * Undelete a file or smart contract that was deleted by SystemDelete; requires a Hedera
- * administrative multisignature. 
+ * Recover a file or contract bytecode deleted from the Hedera File
+ * System (HFS) by a `systemDelete` transaction.
+ *
+ * > Note
+ * >> A system delete/undelete for a `contractID` is not supported and
+ * >> SHALL return `INVALID_FILE_ID` or `MISSING_ENTITY_ID`.
+ *
+ * This transaction can _only_ recover a file removed with the `systemDelete`
+ * transaction. A file deleted via `fileDelete` SHALL be irrecoverable.<br/>
+ * This transaction MUST be signed by an Hedera administrative ("system")
+ * account.
+ *
+ * ### Block Stream Effects
+ * None
  */
 message SystemUndeleteTransactionBody {
     oneof id {
         /**
-         * The file ID to undelete, in the format used in transactions
+         * A file identifier.
+         * <p>
+         * The identified file MUST exist in the HFS.<br/>
+         * The identified file MUST be deleted.<br/>
```

Comment:
```suggestion
```
Delete this line. We are not actually checking that the file was previously deleted.